### PR TITLE
edits to readme

### PR DESCRIPTION
**Pull request by iredwards**
Did a copy edit on the README file as my first pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,9 +191,9 @@ Then the ssh hostnames available will be:
 
 ## Configure your Kubernetes Cluster
 
-Earlier, you copied a sample cluster configuration over into `~/.kraken`  Please take a moment to review the sample configuration and make changes if you desire
+Earlier, you copied a sample cluster configuration over into `~/.kraken`.  Please take a moment to review the sample configuration and make changes if you desire
 
-It may be preferable to save it to a name that is consistent to the `cluster` variable in the configuration.  In other words, if your `cluster` is `foo`, then perhaps your file should be named `foo.yaml`
+You may prefer to save it eith a name that is consistent to the `cluster` variable in the configuration. In other words, if your `cluster` is `foo`, then perhaps your file should be named `foo.yaml`
```
Comment (spiffxp) [inline README.md]: s/eith/with

File: README.md [hunk 2]
```diff
@@ -262,7 +253,7 @@ ip-10-0-58-24.us-west-2.compute.internal     Ready                      3m
 ip-10-0-65-77.us-west-2.compute.internal     Ready                      2m
 ```
 
-#### Getting K8s Deployments
+#### Getting Kubernetess Deployments
```
Comment (spiffxp) [inline README.md]: s/tess/tes/

File: README.md [hunk 3]
```diff
@@ -278,7 +269,7 @@ kube-system   tiller-deploy   1         1         1            1           8m
 
 _Optional step_
 
-In order to feel a bit more confident things are working, try having helm install a new service, like the K8S dashboard
+You can try having helm install a new service, such as the Kubernetess dashboard
```
Comment (spiffxp) [inline README.md]: s/tess/tes/

**Comment by coffeepac** [comment]
LGTM

**Pull request merged**